web_interface/blueprints/api_v3.py

```
@@ -1384,6 +1384,52 @@ def get_system_version():
     except Exception as e:
         return jsonify({'status': 'error', 'message': str(e)}), 500
 
+_update_check_cache: Dict[str, Any] = {'result': None, 'ts': 0.0}
```

Ruff (0.15.13)

[error] Undefined name `Dict`

(F821)